v3/how-to-guides/08-tools/d-subwasm/index.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Verify Runtime Wasm Binaries with Subwasm
+slug: /how-to-guides/v3/tools/subwasm
+keywords: tooling, wasm
+version: 3.0 
+section: how to guides
+category: tools
+---
+
+[Subwasm](https://github.com/chevdor/subwasm) provides a way to expose the features of the runtime for any Substrate chain. 
+This guide steps through the different capabilities that Subwasm provides. 
+
+## Use cases
+
+- Expose verifiable historical information of a chain's runtime for front end applications.
+- Get a quick snapshot of the modules in a runtime.
+- Check for changes between runtimes using a diff.
+- Expose a verifiable runtime for governance transparency.
+
+## Before you begin
+
+To follow this guide you should have a runtime from a chain you wish to query. 
+This could either be one running locally or one running in production.
```

Comment:
To avoid confusion, I took this sentence out. The previous one states that the prerequisite is to have a chain running locally. Don't think it makes sense to specify that the user of this guide needs to be ready to use a chain in production - there's not much preparation needed there.